app/javascript/components/pages/ItemConfirmation.js

```diff
@@ -1,10 +1,88 @@
 import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import { Button, Form, FormGroup, Label, Input } from 'reactstrap';
 
 class ItemConfirmation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      form: {
+        id: undefined,
+        allergies: '',
+        party_id: this.props.party_id,
+        item_bringing: '',
+        user_id: this.props.current_user.id,
+      },
+      submitted: false,
+    };
+  }
+
+  handleChange = (e) => {
+    let { form } = this.state;
+    // console.log('event', e);
+    form[e.target.name] = e.target.value;
+    this.setState({ form: form });
+  };
+
+  handleSubmit = () => {
+    this.props.updateItem(this.state.form, this.state.form.id);
+    this.setState({ submitted: true });
+  };
+
   render() {
+    const unclaimedItems = this.props.items.filter((item) => {
+      return item.user_id === null;
+    });
     return (
       <>
-        <h1>Item Confirmation</h1>
+        <Form inline>
+          <FormGroup className='mb-2 mr-sm-2 mb-sm-0'>
+            <Label for='exampleSelect'>Select</Label>
+            <Input
+              type='select'
+              name='item_bringing'
+              value={this.state.form.id}
+              onChange={(e) => {
+                const item_bringing = unclaimedItems.find(
+                  (item) => item.id == e.target.value
+                ).item_bringing;
+                this.setState({
+                  form: {
+                    ...this.state.form,
+                    id: e.target.value,
+                    item_bringing,
+                  },
+                });
+                // this.handleChange(e)
+              }}
+            >
+              <option>Pick one:</option>
+              {unclaimedItems.map((item) => {
+                return (
+                  <option id={item.id} key={item.id} value={item.id}>
```

Comment:
```suggestion
                  <option key={item.id} value={item.id}>
```
I believe you can probably remove the id attribute because it's not being used by the event handler